src/sx-design/src/SxDesignPortal.js

```diff
@@ -8,20 +8,23 @@ type Props = {
   +children: Node,
 };
 
-export default function TooltipPortal(props: Props): Portal | null {
+const REACT_PORTAL_SELECTOR = '#sx-design-react-portal-root';
```

Comment:
Maybe export it and use it in `src/sx-design/src/SxDesignProvider.js`?

Reply:
Good idea! 👍